### ISSUE TITLE
Add support for Heliosearch HDS distribution of Solr

### ISSUE THREAD

**Issue by apatrida** [kohesive/solr-undertow#14]
Mostly that the --bootstrap command and configuration will support this distribution.  

Depends on #11 for bootstrapping

**Comment by stephlag** [comment]
Heliosearch no longer exists, all the code is now in SolR

**Comment by apatrida** [comment]
This was already done anyway.
